### PR TITLE
Make version number dynamic for "Getting started" guide

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #14159

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](https://github.com/conda/conda/blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

**Review by beeankha — APPROVED** [review]

File: docs/source/conf.py [hunk 1]
```diff
@@ -181,6 +181,9 @@ def setup(sphinx):
     sphinx.connect("autoapi-skip-member", skip_log)
 
 
+# Used to make version number available as a global variable in pages
+rst_epilog = f".. |version| replace:: {version}"
+
```
Comment (kenodegard) [inline docs/source/conf.py:186]: Do we need this? `|version|` and `|release|` are default substitutions already provided: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#substitutions

**Review by kenodegard — CHANGES_REQUESTED** [review]

File: docs/source/user-guide/getting-started.rst [hunk 2]
```diff
@@ -113,9 +113,11 @@ To see your conda version, use the following command::
 
    conda --version
 
-No matter which environment you run this command in, conda displays its current version::
+No matter which environment you run this command in, conda displays its current version:
 
-   conda 23.10.0
+.. parsed-literal::
+
+   conda \ |version|\
```
Comment (kenodegard) [inline docs/source/user-guide/getting-started.rst:120]: Just tested locally and I don't think the `\` are needed

```suggestion
   conda |version|
```

File: docs/source/conf.py (hunk 1 above)
Comment (travishathaway) [reply docs/source/conf.py:186]: Thanks for catching that. Will test without this.

**Comment by travishathaway** [comment]
@kenodegard,

Thanks for catching the unnecessary changes. PR is much cleaner now and ready for another review.

**Review by kenodegard — APPROVED** [review]

**Pull request merged**